File: Sources 4.5.2/Gizmox.WebGUI.Forms/ItemTemplates/CS/CheckBox/CheckBox1.cs
#region Using

using System;
using System.Xml;
using System.Drawing;
using System.ComponentModel;
using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms.Skins;

#endregion Using

namespace $rootnamespace$
{
    /// <summary>
    /// A checkbox control
    /// </summary>
    [Skin(typeof($safeitemname$Skin))]
    [Serializable()]
    public class $safeitemname$ : CheckBox
    {
        public $safeitemname$()
        {
            this.CustomStyle = "$safeitemname$Skin";
        }
    }

}

File: Sources 4.5.2/Gizmox.WebGUI.Forms/ItemTemplates/VB/CheckBox/CheckBox1.vb
Imports System
Imports Gizmox.WebGUI.Forms
Imports Gizmox.WebGUI.Common

' <summary>
' A CheckBox control
' </summary>
<Gizmox.WebGUI.Forms.Skins.Skin(GetType($safeitemname$Skin))> _
<Serializable()> _
Public Class $safeitemname$
Inherits Gizmox.WebGUI.Forms.CheckBox

	Public Sub New()
    	Me.CustomStyle = "$safeitemname$Skin"
	End Sub

End Class
